.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+<!--
+When filing an issue, please fill in the requested fields as thoroughly as you can and then delete this comment
+-->
+
+* Description of issue
+
+    description
```

Comment:
I suggest that anything that needs to be replaced by the user, put it inside <brackets> and be very explicit / foolproof.
For example:

<Replace with a description of the issue>
<Replace with the output of the above command's execution>
etc